pkg/test/tier/templateRefs.go

```diff
@@ -8,14 +8,11 @@ import (
 	"github.com/codeready-toolchain/toolchain-common/pkg/hash"
 )
 
-// ComputeTemplateRefsHash computes the hash of the `.spec.namespaces[].templateRef` + `.spec.clusteResource.TemplateRef`
+// ComputeTemplateRefsHash computes the hash of the value of `status.revisions[]`
 func ComputeTemplateRefsHash(tier *toolchainv1alpha1.NSTemplateTier) (string, error) {
 	var refs []string
-	for _, ns := range tier.Spec.Namespaces {
-		refs = append(refs, ns.TemplateRef)
-	}
-	if tier.Spec.ClusterResources != nil {
-		refs = append(refs, tier.Spec.ClusterResources.TemplateRef)
+	for _, rev := range tier.Status.Revisions {
+		refs = append(refs, rev)
 	}
 	sort.Strings(refs)
 	m, err := json.Marshal(templateRefs{Refs: refs})
```

Comment:
I know that you didn't introduce these functions, but I'm wondering if there is any difference compared to the functions that are in the `pkg/hash`. Couldn't we remove them?

Reply:
I did notice it, was waiting to remove the refernece first from e2e and then remove it from here.. but as discussed, i have removed it now .